dev_scripts/update_pt_data.py

```diff
@@ -123,7 +123,7 @@ def parse_radii():
             print(el)
     with open("periodic_table2.yaml", mode="w") as file:
         yaml.dump(data, file)
-    with open("../pymatgen/core/periodic_table.json", mode="w") as file:
+    with open(f"{PKG_DIR}/core/periodic_table.json", mode="w") as file:
         json.dump(data, file)
```

Comment:
I would not change `open` for `json.dump` at this moment, as [`json.dump` would escape non-ASCII chars by default](https://docs.python.org/3/library/json.html#json.dump):
> If ensure_ascii is true (the default), the output is guaranteed to have all incoming non-ASCII characters escaped. If ensure_ascii is false, these characters will be output as-is.

Reply:
then let's put the encoding explicitly

Reply:
Oh yes of course, sorry I missed this one, thanks for catching this!